### ISSUE TITLE
protobuf support

### ISSUE THREAD

**Issue by nikoncode** [vladmihalcea/hibernate-types#368]
We are using protobuf to store data. So are you planning to add support for protobuf to be (de)serialized into jsonb? It's easy to implement.

**Comment by vladmihalcea** [comment]
At the moment, I don't have time to work on OSS in my spare time. So, there are two ways this feature could be implemented:

1. You can implement it yourself and provide a Pull Request. Make sure to add it to all modules and provide test cases.
2. Or, I could do it via consulting.

**Comment by nikoncode** [comment]
@vladmihalcea I will implement my idea inside my project and if it will work will try to return code into `hibernate-types`.

just wondering does `hibernate-types` support hibernate 5.6.x?

**Comment by vladmihalcea** [comment]
> I will implement my idea inside my project and if it will work will try to return code into hibernate-types.

You can fork this repository, implement the idea, test it on your project and send it as a Pull Request.

> just wondering does hibernate-types support hibernate 5.6.x?

Most likely, yes. Otherwise, [this issue](https://github.com/vladmihalcea/hibernate-types/issues/367) would have shown a problem, rather than just a Hibernate deprecation warning.

**Comment by nikoncode** [comment]
@vladmihalcea it's quite hard to start working because of missing contribution guide. 

Usually when I am trying to implement something for opensource I am starting from clean install, but in this case it's not really possible because I have all tests failed.

**Comment by vladmihalcea** [comment]
Of course, it's possible. Otherwise, there wouldn't have been any contributions.

Your tests are failing because the DB settings don't match your environment. 

No one had the time to implement a contribution guide or set up Testcontainers to ease the ramp-up process. You can start with that. Adding Testcontainers is an easy task and will fix your issue.

**Comment by peyerroger** [comment]
Hi @nikoncode and @vladmihalcea .

We're also interested in storing protobuf as jsonb columns in postgres. I'm happy to help out here, but I'm a bit lost at this point. So any advice how to tackle this issue would be greatly appreciated.

@nikoncode Have you already a somewhat working version of this feature? If yes, what's missing to push it over the finish line?

Thanks for your answers,
Roger

**Comment by vladmihalcea** [comment]
@peyerroger There are plenty of test cases in the repository you can run to see how the existing Types are working.
